rotkehlchen/tests/unit/decoders/test_pendle.py

```diff
@@ -539,3 +540,329 @@ def test_swap_using_odos(ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x3fEB789a337E7C9E82d6E1AB16f1C9AC116c6956']])
+def test_mint_py(ethereum_inquirer, ethereum_accounts):
```

Comment:
```suggestion
def test_mint_pt_yt(ethereum_inquirer, ethereum_accounts):
```
Same for `test_redeem_py` - since `sy` in `test_mint_sy` refers to a single token it looks like `py` is supposed to refer to a single token as well, so should probably use both the `pt` and `yt` abreviations that are used elsewhere to avoid confusion.